starter-code/javascript/chronometer.js

```diff
@@ -27,9 +37,11 @@
 
 // };
 
-// Chronometer.prototype.stopClick = function () {
+Chronometer.prototype.stopClick = function () {
+
+    this.intervalId = window.clearInterval(this.currentTime);
```

Comment:
For clear an interval you have to pus in the argument the id of the interval. And you don't need to save the return value. So you should have something like:
```
window.clearInterval(this.intervald);
```